File: python/s01-ec-01-age-cap.py
#!/usr/bin/env python3

"""Ecrivez un programme age.cc qui :

    demande son age a l'utilisateur ;
    lit la reponse de l'utilisateur et l'enregistre dans une variable age de type entier;
    calcule l'annee de naissance (a un an pres) de l'utilisateur et l'enregistre dans la variable annee de type entier;
    affiche l'annee de naissance ainsi calculee.
"""

YEAR = 2016;

age = input("what your age ? ")
birthDate = YEAR - age;
print("your are born in " + str(birthDate));

File: c/s01-ec-01-age-cap.c
/*Écrivez un programme age.cc qui :

    demande son âge à l'utilisateur ;
    lit la réponse de l'utilisateur et l'enregistre dans une variable age de type entier ;
    calcule l'année de naissance (à un an près) de l'utilisateur et l'enregistre dans la variable annee de type entier ;
    affiche l'année de naissance ainsi calculée.
*/

#include <stdlib.h>
#include <stdio.h>

int main ()
{
	const int year =2016;
	int age = 0;
	printf("what your age ? ");
	scanf("%d", &age);
	
	int birthDate = year - age;
	printf("your are born in %d", birthDate);
	
	return 0;
}
